designer/client/conditions/ConditionsEdit.tsx

```diff
@@ -23,7 +23,7 @@ function useConditionsEditor() {
     setShowAddCondition(false);
   }
   function cancelInlineCondition(e) {
-    e?.preventDefault();
+    if (e?.preventDefault) e.preventDefault();
```

Comment:
e?.preventDefault() is checking if e is not `null` or `undefined`, then calls preventDefault(). If it is null/undefined, it won't continue the chain. e?.preventDefault() should be enough - but if you need to check if the method exists as well you can do `e?.preventDefault?.()`

Reply:
this method is called from button event handler too - yup i will change it to concise version